src/java/org/apache/cassandra/io/sstable/format/SSTableReader.java

```diff
@@ -1645,6 +1645,12 @@ public DecoratedKey keyAt(long indexPosition) throws IOException
         }
     }
 
+    public DecoratedKey keyAt(RandomAccessReader reader, long position) throws IOException
```

Comment:
Unused method

Reply:
...because we don't use offsets anymore?